Documentation/packaging.md

```diff
@@ -81,6 +55,35 @@ Packages using the `--with-stage1-flavors=host` option must therefore add a run-
 Whenever systemd and bash are upgraded on the host, rkt will use the new version at run time.
 It becomes the packager's responsibility to test the rkt package whenever a new version of systemd is packaged.
 
-# Other code bundle
+### Godep
+
+rkt uses [Godep](https://github.com/tools/godep) to maintain [a copy of dependencies in its source repository](https://github.com/coreos/rkt/tree/master/Godeps).
+
+## Run-time dependencies
+
+A Linux system configured with [suitable options](hacking.md#run-time-requirements) is required.
+
+### Additional run-time dependencies for the host flavor
+
+- systemd >= v222 with systemctl, systemd-shutdown, systemd, systemd-journald.
+- bash
+
+## Packaging Externals
+
+### Ownership and permissions of rkt directories
+
+In general, subdirectories of `/var/lib/rkt` should be created with the same ownership and permissions as if created by `rkt install`, see [directory list](https://github.com/coreos/rkt/blob/master/rkt/install.go#L44).
+
+Any rkt package should create a system group `rkt`, and `/var/lib/rkt` should belong to group `rkt` with the `setgid` bit set (`chmod g+s`)
+
+When the ownership and permissions of `/var/lib/rkt` are set up correctly, members of group `rkt` should be able to fetch ACIs without using `sudo`. Root privilege is still required to run pods.
+
+### systemd units
+
+A few [example systemd unit files for rkt helper services][rkt-units] are included in the rkt sources. These units demonstrate a systemd-managed, socket-activated rkt [metadata-service][rkt-metadata-svc], along with a convenient periodic [garbage collection][rkt-gc] service invoked at 12-hour intervals to purge dead pods.
```

Comment:
WIP for rkt api-service units: https://github.com/joshix/rkt/tree/apisvcunit/dist/init/systemd